settings.gradle

```diff
@@ -14,18 +14,3 @@ dependencyResolutionManagement {
 }
 rootProject.name = "polypoly"
 include ':app'
-
-ext.isCiServer = System.getenv().containsKey("CIRRUS_CI")
```

Comment:
Thanks for the PR :)
These lines should be kept for the CI to run correctly, and the PR is also missing the `.cirrus.yml` and `.codeclimate.yml` files.